src/include/food.hpp

```diff
@@ -5,25 +5,39 @@
 #include <SFML/Window.hpp>
 
 
+enum class FoodType : std::size_t 
+{
+    Neutral = 0U,
+    Aggressive,
+    Valuable
+};
+
+
 class Food : public sf::Drawable {
   public:
-    explicit Food() = default;
+    explicit Food();
 
-    explicit Food(const sf::Vector2f position);
+    explicit Food(const sf::Vector2f position, const sf::Vector2f size);
```

Comment:
AFAIK, there's no need in `explicit` if the constuctor takes more than one argument.

Reply:
Yes, thanks.